### PR TITLE
add Firely Terminal to image

### PR DESCRIPTION
**Pull request by 11011san**
add .net 6  sdk and dependences.
add Firely Terminal

### CODE REVIEW

**Review by peturv — APPROVED** [review]
👍

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +21,19 @@ RUN /bin/sh -c set -eux; apk add --update \
     openjdk11 \
     fontconfig \
     ttf-dejavu \
-    && rm -rf /var/cache/apk/*
+    icu-libs \
+    krb5-libs \
+    libgcc \
+    libintl \
+    libintl \
+    libintl \
```
Comment (vadi2) [inline Dockerfile:29]: This seems repeated

**Review by vadi2 — APPROVED** [review]

**Comment by peturv** [comment]
The image is now available as : `docker.io/chorusab/fsh-fhir-build-image:latest`
I does not seem to push the actual build tag `docker.io/chorusab/fsh-fhir-build-image:0.9.1709047262` in my action, I need to have a look at that. @vadi2 @11011san

File: Dockerfile (hunk 1 above)
Comment (peturv) [reply Dockerfile:29]: You are a lot more alert than I am today Vadim :) 
Cosmetic defect that I'll clean up next time I need to change the project. I need to correct the bush of build tags soon anyways.

**Pull request merged**